components/Loading/styles.ts

```diff
@@ -0,0 +1,60 @@
+import { keyframes } from '@stitches/react';
+import { styled } from '../../styles/stitches.config';
+
+const dots = keyframes({
+    '0% ': {
+        backgroundColor: '#FF6680'
+    },
+
+    '100%': {
+        backgroundColor: '#FFFFFF'
+    }
+})
+
+export const LoadingContainer = styled('div', {
+    width: '100%',
```

Comment:
poderia ser 100vw né?

Reply:
Poderia.. tem alguma vantagem?

Reply:
Se esse componente não for filho de nenhum outro, tem uma diferença mais não necessariamente uma vantagem. Pensei que como o height tá com vh, aqui poderia ser vw, mas pode deixar :)